src/writer/proposal.ts

```diff
@@ -104,22 +104,19 @@ export async function verify(body): Promise<any> {
     if (msg.payload.type !== space.voting.type) return Promise.reject('invalid voting type');
   }
 
-  /**
   try {
     const content = `
       ${msg.payload.name || ''}
       ${msg.payload.body || ''}
       ${msg.payload.discussion || ''}
     `;
-
     if (await isMalicious(content)) {
-      blockaidBlockedRequestsCount.inc({ space: space.id });
+      chainpatrolBlockedRequestsCount.inc({ space: space.id });
```

Comment:
Maybe move this one directly inside the chainpatrol helper

Reply:
no `space.id` in helper 🙈 is it worth to pass?

Reply:
there is no `space` name inside `proposal` data 😄

Reply:
will pass `space` as seperate param for now